content/applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst

```diff
@@ -62,59 +58,57 @@ second work center when the first is unavailable for any reason.
 Add time off for a work center
 ==============================
 
-With configuration completed, time off can now be assigned to the work center that will be made
-unavailable. Begin by navigating to :menuselection:`Manufacturing --> Configuration --> Work
-Centers` and selecting the affected work center. Click :guilabel:`Edit`, and then the :guilabel:`↗
-(external link)` button next to the :guilabel:`Working Hours` drop-down menu.
+With configuration completed, time off can now be assigned to a work center. Begin by navigating to
+:menuselection:`Manufacturing app --> Configuration --> Work Centers`, and selecting the affected
+work center. Click the :icon:`oi-arrow-right` :guilabel:`Internal link` button on the right side of
+the :guilabel:`Working Hours` drop-down menu, to open the working hours page for the work center.
 
 .. image:: work_center_time_off/working-hours-button.png
    :align: center
    :alt: The Working Hours "External link" button on the work center form.
 
-A pop-up appears, titled :guilabel:`Open: Working Hours`. The standard working hours for the work
-center are listed here, along with various other details about it. Since developer mode was enabled,
-there is a :guilabel:`Time Off` button in the top right of the pop-up. Click it to be taken to the
-:guilabel:`Resource Time Off` page.
-
-.. image:: work_center_time_off/time-off-button.png
-   :align: center
-   :alt: The Time Off button on the Working Hours pop-up.
+The working hours page displays the standard working hours for the work center. With developer mode
+activated, a :icon:`fa-plane` :guilabel:`Time Off` smart button appears at the top of the page.
+Click it to open the :guilabel:`Resource Time Off` page.
 
-On this page, click :guilabel:`Create` to configure a new time-off entry. On the time-off form, note
-the :guilabel:`Reason` for the work center closure (broken, maintenance, etc.), select the affected
-work center as the :guilabel:`Resource`, and choose a :guilabel:`Start Date` and :guilabel:`End
-Date` to specify the period during which the work center will be unavailable. Click :guilabel:`Save`
-and the time off for the work center is logged in Odoo.
+On this page, click :guilabel:`New` to configure a new time-off entry. On the time-off form, note
+the :guilabel:`Reason` for the work center closure (broken equipment, maintenance, etc.), select the
+affected work center as the :guilabel:`Resource`, and choose a :guilabel:`Start Date` and
+:guilabel:`End Date` to specify the period during which the work center will be unavailable.
 
```

Comment:
do you feel like it would be beneficial to mention the 'Time Off Request' field, Working Hourse field, Work Entry Type field...?

Reply:
No, those fields aren't specific to this workflow.